lib/oss/version.rb

```diff
@@ -1,3 +1,3 @@
 module SlideHub
-  VERSION = '1.0.0'
+  VERSION = '1.0.1'
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.